setup.py

```diff
@@ -1,19 +1,12 @@
 from setuptools import setup
-try:
-    import multiprocessing
-except ImportError:
-    pass
+from httpie_aws_authv4 import __version__
```

Comment:
The side affect of creating a dependency on `httpie_aws_authv4` is that we can't do a clean install using `python setup.py install` if `httpie` isn't already installed (at least in my environment).  It seems this might be a [common problem though](https://packaging.python.org/guides/single-sourcing-package-version/).

Reply:
Thanks for the link presenting various solutions.  I chose to just read in the main file and search for the version using a RegEx.

Reply:
Looks good, just need to remove line 7 as well

Reply:
Good catch!